packages/chakra-ui/src/components/ui/close-button.tsx

```diff
@@ -0,0 +1,14 @@
+import type { ButtonProps } from '@chakra-ui/react';
+import { IconButton as ChakraIconButton } from '@chakra-ui/react';
+import * as React from 'react';
+import { LuX } from 'react-icons/lu';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { forwardRef } from 'react'`;